control/vehicle_cmd_gate/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 3.5)
+project(vehicle_cmd_gate)
+
+### Compile options
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  # add_compile_options(-Wall -Wextra -Wpedantic)
+  add_compile_options(-Wno-unused-parameter)
+endif()
+
+### Dependencies
+find_package(ament_cmake REQUIRED)
+find_package(autoware_control_msgs REQUIRED)
+find_package(autoware_vehicle_msgs REQUIRED)
+find_package(rclcpp REQUIRED)
+
+# Add path of include dir
+include_directories(include)
```

Comment:
I think CMake recommends using `target_include_directories` rather than a global include directories

Reply:
I see that Steve is also using `include_directories` here https://github.com/ros-planning/navigation2/blob/main/nav2_costmap_2d/CMakeLists.txt#L32-L35. So I would leave this as is.

Reply:
:+1: Done